examples/kubernetes-local-redirect/node-local-dns-lrp.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumLocalRedirectPolicy
+metadata:
+  name: "nodelocaldns-lrp"
```

Comment:
you can remove `-lrp` since it's obvious from the `kind`

Reply:
Done.